core/templates/dev/head/components/summary_tile/collection_summary_tile_directive.html

```diff
@@ -1,12 +1,40 @@
-<md-card class="oppia-activity-summary-tile">
+<span ng-if="learnerDashboardActivityIds">
+<i class="protractor-test-add-exp-to-playlist-icon oppia-learner-dashboard-icon fa fa-clock-o"
```

Comment:
I tried this, and I don't think this would be a good idea as this piece is dynamic and dependent on the parent. So I'm facing a lot of trouble in interfacing this. Especially the hovering part.

Reply:
If you had a service that was directly injected into the directive and provided the playlist data, then the directive can take care of its own hovering state, right? Not sure why the parent needs to know...

Reply:
Done!